spec/forms/waste_exemptions_engine/contact_address_manual_form_spec.rb

```diff
@@ -4,13 +4,130 @@
 
 module WasteExemptionsEngine
   RSpec.describe ContactAddressManualForm, type: :model do
-    it_behaves_like "a manual address form", :contact_address_manual_form
```

Comment:
The code in this spec will be need to be moved back in a shared example once the rest of the forms are in the same state.